src/components/Canvas/CanvasBuilder.jsx

```diff
@@ -35,20 +47,6 @@ export class CanvasBuilder {
     return this
   }
 
-  /**
-   * @returns {number}
-   */
-  get height () {
-    return this.#height
-  }
-
-  /**
-   * @returns {number}
-   */
-  get width () {
-    return this.#width
-  }
```

Comment:
Removed these getters. I don't think we'll ever need to access height and width from outside the class.